src/pages/blog/2020-airship-tc-election.md

```diff
@@ -0,0 +1,48 @@
+---
+templateKey: blog-post
+title: Congratulations To The 2020-2021 Technical Committee
+author: Alexander Hughes
+date: 2020-06-22T12:00:00.000Z
+category: 
+  - id: category-A7xLnYrE1
+    label: Election
+---
+
+The Airship team is excited to announce the completion of its second [_Technical Committee election_](
+http://lists.airshipit.org/pipermail/airship-discuss/2020-June/001005.html). The [_Technical Committee_](
+https://opendev.org/airship/governance#user-content-technical-committee), one of the two governing bodies for Airship,
+is responsible for ensuring Airship projects adhere to the project's core principles, promote standardization, and
+define and organize the Airship versioning and release process.
+
+The Airship Technical Committee is composed of five elected members, representing leaders from five different companies:
+
+* Alex Bailey, AT&T
+* Alexander Hughes, Accenture
+* Alexey Odinokov, Mirantis
+* Jan-Erik Mångs, Ericsson
+* John (JT) Williams, Dell
+
+<br>
+
+We want to extend our congratulations to the new Technical Committee members and recognize the contributions of Andrew
+Leasck, James Gu, and Ryan van Wyk to the Airship Community through their roles on the Technical Committee last year.
+Governance by community-elected officials reflects Airship's continued dedication to the [_Four Opens_](
+https://osf.dev/four-opens/). The Technical Committee is organizing its schedule for the upcoming year. Once finalized,
+the Technical Committee will share the meeting schedule for 2020-2021 on the [_Wiki_](
+https://wiki.openstack.org/wiki/Airship/Airship-TC), where previous meeting agendas and recordings are also available.
+
+The Technical Committee is just one of the two governing bodies within the Airship community. With that election wrapped
```

Comment:
The first sentence here is a bit redundant, you mention this above.  I'd start this paragraph with "With the Technical Committee election wrapped up, they have turned..."

Reply:
Updated.